### ISSUE TITLE
scope.value.values can be null

### ISSUE THREAD

**Issue by JimBobSquarePants** [umco/umbraco-vorto#84]
I've seen this error pop up a few times when I have empty Vorto values in a doctype and I save or publish a document type.

> TypeError: Cannot set property 'en-AU' of null

The error is thrown in the function below.
```
var unsubscribe = scope.$on("vortoSyncLanguageValue", function (ev, args) {
                if (args.language === scope.language) {
                    scope.$broadcast("formSubmitting", { scope: scope });
                    scope.value.values[scope.language] = scope.model.value; // Error thrown here
                }
            });
```

It looks like you need to ensure values is always an array but maybe you know why is could ever be.

**Comment by Jitse** [comment]
I think this was already fixed in commits 97d9020 and 075b9cf on November 1st, it just hasn't been released yet.

**Comment by mattbrailsford** [comment]
Appears to be fixed
